app/src/app/[lng]/organization/[id]/account-settings/account/account-settings-tab.tsx

```diff
@@ -166,7 +217,55 @@ const AccountSettingsTab = ({ t }: { t: TFunction }) => {
         >
           {t("brand-settings")}
         </Tabs.Trigger>
+        <Tabs.Trigger
+          value="manage-password"
+          fontFamily="heading"
+          justifyContent={"left"}
+          letterSpacing={"wide"}
+          color="content.secondary"
+          lineHeight="20px"
+          fontStyle="normal"
+          fontSize="label.lg"
+          height="52px"
+          w={"223px"}
+          _selected={{
+            color: "content.link",
+            fontSize: "label.lg",
+            fontWeight: "medium",
+            backgroundColor: "background.neutral",
+            borderRadius: "8px",
+            borderWidth: "1px",
+            borderStyle: "solid",
+            borderColor: "content.link",
+          }}
```

Comment:
This (and the other) styling could be extracted to a variant of the tab component in our theme or as a Chakra recipe of it.